### ISSUE TITLE
Coordinate Mode does not properly support multiple monitors (windows)

### ISSUE THREAD

**Issue by 400Ferrets** [MrTanoshii/rusty-autoclicker#24]
Coordinate selector properly addresses monitors when they are to the right of or above the primary monitor, but when secondary monitors are to the left of or below the primary monitor the selector will properly display coordinates following a grid with its origin located at the bottom left of the primary monitor, with negative values, however the coordinates will be filled in the main window as their absolute value (- removed). Additionally, when auto-clicking the program will incorrectly move the mouse to the coordinate position assuming the origin is located furthest left and down of the monitor setup, instead of the bottom left of the primary monitor as it should.

I tried to make this as clear as possible, but did a poor job. please ask for clarification if necessary.

TL;DR
when main display is located bottom left of all displays, things work correctly
when main display is located right or above secondary display(s), things do not work correctly, as the Coord picker does not fill negative values, and the mouse position uses an incorrect origin.

**Comment by MrTanoshii** [comment]
Good catch! The issue seems to be with the string sanitization

**https://github.com/MrTanoshii/rusty-autoclicker/blob/9afe987a166bec7925af5833cdc24d6c6eaafb13/src/app.rs#L227-L228**

**Comment by MrTanoshii** [comment]
Sorry for the delays!
Thanks for bringing the issue to our attention; it should be fixed in https://github.com/MrTanoshii/rusty-autoclicker/releases/tag/v2.2.0

Please reopen the issue if it's not fixed on your end.
